assets/extensions/card.js

```diff
@@ -8,32 +8,37 @@ import { __ } from '@wordpress/i18n';
  */
 import ExtensionActions from './extension-actions';
 
+// prettier-ignore
+// const extensionMock = {"hash":"d1a69640d53a32a9fb13e93d1c8f3104","title":"WooCommerce Paid Courses","image":null,"excerpt":"Sell your courses using the most popular eCommerce platform on the web \u2013 WooCommerce.","link":"https:\/\/senseilms.com\/product\/woocommerce-paid-courses\/","price":"$129.00","is_featured":false,"product_slug":"sensei-wc-paid-courses","hosted_location":"external","type":"plugin","plugin_file":"woothemes-sensei\/woothemes-sensei.php","wccom_product_id":"152116"};
+// prettier-ignore
+// const extensionMock = {"version":"5.0.0.0.0.0","hash":"d1a69640d53a32a9fb13e93d1c8f3104","title":"WooCommerce Paid Courses","image":null,"excerpt":"Sell your courses using the most popular eCommerce platform on the web \u2013 WooCommerce.","link":"https:\/\/senseilms.com\/product\/woocommerce-paid-courses\/","price":"$129.00","is_featured":false,"product_slug":"sensei-wc-paid-courses","hosted_location":"external","type":"plugin","plugin_file":"woothemes-sensei\/woothemes-sensei.php","wccom_product_id":"152116"};
+// prettier-ignore
+const extensionMock = {"version":"5.0.0.0.0.0","has_update":true,"hash":"d1a69640d53a32a9fb13e93d1c8f3104","title":"WooCommerce Paid Courses","image":null,"excerpt":"Sell your courses using the most popular eCommerce platform on the web \u2013 WooCommerce.","link":"https:\/\/senseilms.com\/product\/woocommerce-paid-courses\/","price":"$129.00","is_featured":false,"product_slug":"sensei-wc-paid-courses","hosted_location":"external","type":"plugin","plugin_file":"woothemes-sensei\/woothemes-sensei.php","wccom_product_id":"152116"};
+
 /**
  * Extensions card component.
  *
  * @param {Object}  props           Component props.
- * @param {boolean} props.hasUpdate Whether extensions has update.
+ * @param {boolean} props.extension Extension object.
  */
-const Card = ( { hasUpdate } ) => (
+const Card = ( { extension = extensionMock } ) => (
```

Comment:
Temporary approach to use the mock until we integrate the data and decide how to display it.